dotcom-rendering/src/web/components/FilterLink.tsx

```diff
@@ -76,51 +76,45 @@ const Label = ({ value, count }: LabelProps) => (
 );
 
 /**
- * # Filter Button
+ * # Filter Link
  *
  * A child of `TopicFilterBank`
  *
- * ## Why does this need to be an Island?
- *
- * ⚠️ It does not need to be! ⚠️
- *
- * (It is embedded within `TopicFilterBank`)
- *
  * ---
  *
  * [`FilterButton` on Chromatic](https://www.chromatic.com/component?appId=63e251470cfbe61776b0ef19&csfId=components-filterbutton)
```

Comment:
given that the stories have now changed I'm guessing this can go / be renamed to FilterLink